### DISCUSSION TITLE
Track another branch of dependent repository

### DISCUSSION THREAD

**Discussion by ashubisht** [renovatebot/renovate#13900] [Request Help]
HI Community,

I have two Golang repositories where one is dependent on another (Say Repo A is dependent on Repo B)
I want to track changes of Repo B and update Repo A, based on it.
However, I want to track another branch named dev of Repo B (instead of main branch), based on its commits. Is it possible to do the same?

Currently I am using pseudo version in Repo A, and while Renovate bot does brings changes based on release tags on Repo B, I am wondering if Renovate Bot can help us bringing dev changes of Repo B.

**Comment by rarkins** [comment]
I'm not sure we support this today. If you can create a public reproduction of this scenario then we can take a closer look

**Reply by ashubisht** [reply]
Sure, I will create a repository and provide link for the same. I will need a day. Thanks for the answer!
